test/unit/file_transfer_agent/file_transfer_agent_test.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+const assert = require('assert');
+const testUtil = require('../../integration/testUtil');
+const os = require('os');
+const fs = require('fs');
+const crypto = require('crypto');
+const getMatchingFilePaths = require('./../../../lib/file_transfer_agent/file_transfer_agent').getMatchingFilePaths;
+
+
+function clean(randomName) {
+  const matchedFiles = getMatchingFilePaths(os.tmpdir(), `${randomName}matched` + '*', 'win32');
+  const notmatchedFiles = getMatchingFilePaths(os.tmpdir(), `${randomName}notmatched` + '*', 'win32');
+
+  matchedFiles.forEach(filePath => {
+    fs.rmSync(filePath);
```

Comment:
can we use async api? (but it's not blocking)

Reply:
ok